templates/flows/flow_launch.haml

```diff
@@ -0,0 +1,267 @@
+    
+    -if warnings
+      %div.warning
+        -for warning in warnings
+          %p= warning
+
+        %p
+          -trans "You may still start this flow but WhatsApp contacts who have not sent an incoming message in the last 24 hours may not receive it."
+    
+    %script{src:"{% url 'django.views.i18n.javascript_catalog' %}"}
+    %script{src:"{{ STATIC_URL }}js/schedules-ui.js?v=3"}
+    :javascript
+      var user_tz = "{{ user_tz }}";
+      var user_tz_offset = "{{ user_tz_offset }}";
+      var launchSelector = $('#id_launch_type');
+      var startFlowTypeSelector = $('#id_start_type');
+      var launchImmediatelyBox = $('.l-immediately');
+      var launchOnKeywordTriggerBox = $('.l-keyword');
+      var launchOnScheduleTriggerBox = $('.l-schedule');
+      var repeatPeriodSelector = $('#id_repeat_period');
+
+      launchSelector.select2({containerCssClass: "select2-temba-field", minimumResultsForSearch: -1});
+      startFlowTypeSelector.select2({containerCssClass: "select2-temba-field", minimumResultsForSearch: -1});
+      repeatPeriodSelector.select2({containerCssClass: "select2-temba-field", minimumResultsForSearch: -1});
+
+      function showFieldsOnStartType(selected) {
+        let omnibox = $(".omnibox");
+        let query = $(".contact-query");
+        if (selected === 'query') {
+          omnibox.hide();
+          query.show();
+        } else {
+          query.hide();
+          omnibox.show();
+        }
+      }
+      
+      showFieldsOnStartType(startFlowTypeSelector.val());
+      startFlowTypeSelector.change(function (){showFieldsOnStartType($(this).val());});
+      
+      function showLaunchFlowBox(selected) {
+        launchImmediatelyBox.hide();
+        launchOnKeywordTriggerBox.hide();
+        launchOnScheduleTriggerBox.hide();
+        switch(selected) {
+          case "LAUNCH_IMMEDIATELY":
+            launchImmediatelyBox.show();
+            showFieldsOnStartType(startFlowTypeSelector.val());
+            break;
+          case "LAUNCH_ON_KEYWORD_TRIGGER":
+            launchOnKeywordTriggerBox.show();
+            break;
+          case "LAUNCH_ON_SHEDULE_TRIGGER":
+            launchOnScheduleTriggerBox.show();
+            break;
+        }
+      };
+
+      showLaunchFlowBox(launchSelector.val());
+      launchSelector.change(function() {showLaunchFlowBox($(this).val())});
+
+      $('#id_match_type').select2({containerCssClass: "select2-temba-field", minimumResultsForSearch: -1});
+      $('#id_groups').select2({
+        containerCssClass: "select2-temba-field"
+      });
+
+      $(document).ready(function() {
+        var minDate = new Date();
+        var minutes = minDate.getMinutes();
+        if (minutes > 0) {
+            minDate.setMinutes(0);
+            minDate.setHours(minDate.getHours() + 1);
+        }
+        initializeDatetimePicker(minDate, undefined, false);
```

Comment:
The appearance of this `datetime` picker should be like this attached screenshot and you can find references here https://github.com/greatnonprofits-nfp/rapidpro/blob/release/0.22/templates/flows/flow_launch_schedule.haml and here 
https://github.com/greatnonprofits-nfp/rapidpro/commit/09be08924731f7837001c09f08ced9f690736a58#diff-2cb5ce8cbba47341d8f7a115565fdc1f

![Screen Shot 2020-04-16 at 12 23 06](https://user-images.githubusercontent.com/5198870/79475159-b4709b00-7fdd-11ea-8677-3bd3fa57d9e2.png)